waku/factory/waku.nim

```diff
@@ -374,16 +374,6 @@ proc startWaku*(waku: ptr Waku): Future[Result[void, string]] {.async: (raises:
   if not waku[].deliveryMonitor.isNil():
     waku[].deliveryMonitor.startDeliveryMonitor()
 
-  ## libp2p DiscoveryManager
-  waku[].discoveryMngr = DiscoveryManager()
-  waku[].discoveryMngr.add(
-    RendezVousInterface.new(rdv = waku[].node.rendezvous, tta = 1.minutes)
-  )
-  if not isNil(waku[].node.wakuRelay):
-    for topic in waku[].node.wakuRelay.getSubscribedTopics():
-      debug "advertise rendezvous namespace", topic
-      waku[].discoveryMngr.advertise(RdvNamespace(topic))
```

Comment:
I don't understand. How is rendezvous related to circuit-relay?

Reply:
Yes you are absolutely right that are different concepts. Nevertheless, given that this snippet was part of the circuit-relay PR, I think it worth double-checking it still works.

Besides, it is curious that we are not instantiating the `DiscoveryManager` type in this PR so maybe we are missing it :)

Reply:
I will double check what `DiscoveryManager` does.

I did not use it because it has a very limited API that prevented me from impl. our rendezvous strategy.